tutorials/NWB Tutorials/read_in_NWB.m

```diff
@@ -0,0 +1,49 @@
+%% *MatNWB Setup*
+% Start by setting up your MATLAB workspace. The code below adds the directory 
+% containing the MatNWB package to the MATLAB search path. MatNWB works by automatically 
+% creating API classes based on a defined schema. Running the generateCore() function
+% generates these classes for the lastest schema version.
+
+path_to_matnwb = '~/Repositories/matnwb';% change to your own path location
+addpath(genpath(pwd));
+generateCore();
+%% *Load in Image Files*
+% The extractor.m function function requires two inputs
+
+%  1) An image stack with dimensions [rows, columns, frames]
+%  2) A config strcuture with the parameters for the EXTRACT algorithm
+
+
+% You can read in an nwb file with the nwbRead function from MatNWB. The code 
+% below reads in a file downloaded from this DANDIset <https://gui.dandiarchive.org/#/dandiset/000054 
+%For this file, the raw images are located as a DataStub within 
+% the TwoPhotonSeries object. In MatNWB, the data is read "lazily". This means 
+% that, instead of reading the entire dataset into memory, you have the option 
+% of reading just a subset of the data stored on disk. The code below reads into 
+% memory only the first 2,000 frames of the data. You can refer to the MatNWB 
+% ophys tutorial to learn more about the structure and contents of NWB files. 
+
+% Load File (change to your own file location)
+inputFilePath = '/Users/cesar/Documents/DANDI_files/sub-F1_ses-20190407T210000_behavior+ophys.nwb';
+nwb = nwbRead(inputFilePath);
+% Load in image stack
+image_stack = nwb.acquisition.get('TwoPhotonSeries').data.internal.stub(:, :, 1:2000);
```

Comment:
Thanks for sorting this out. I think it would be nice if `nwb.acquisition.get('TwoPhotonSeries').data(:, :, 1:2000)` worked, but that's obviously an issue for MatNWB, not here. Would you mind opening an issue for this?